warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/ShardedTableMapFile.java

```diff
@@ -334,7 +335,7 @@ public static Path createShardedMapFile(Logger log, Configuration conf, Path wor
      */
     public static Map<Text,String> getLocations(Logger log, AccumuloHelper accumuloHelper, String shardedTableName) {
         // split (endRow) -> String location mapping
-        Map<Text,String> splitToLocation = new TreeMap<>();
+        TreeMap<Text,String> splitToLocation = new TreeMap<>(Comparator.nullsLast(Comparator.naturalOrder()));
```

Comment:
Does the order of nulls really matter if we're preventing nulls from going in the map?

Reply:
Please remove this line, as it is pointless (we don't need a comparator that orders nulls last if the comparator will never compare any nulls since none can go in the treemap).